frontend/utilities/file/fileUtils.ts

```diff
@@ -0,0 +1,34 @@
+type IPlatformDisplayName = "macOS" | "Windows" | "linux";
+
+const getFileExtension = (file: File) => {
+  const nameParts = file.name.split(".");
+  return nameParts.slice(-1)[0];
+};
+
+export const FILE_EXTENSIONS_TO_PLATFORM_DISPLAY_NAME: Record<
```

Comment:
I'd probably just simplify this with a map of extensions->platform:

```ts
const PLATFORM_BY_EXTENSION: Record<string, IPlatformDisplayName> = {
  json: "macOS",
  pkg: "macOS",
  ...
  deb: "linux",
} as const;

export const getPlatformDisplayName = (file: File) => {
  return PLATFORM_BY_FILE_EXTENSION[getFileExtension(file)] ?? "";
};

Reply:
updated